recipes/osx-sysroot/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "osx-sysroot" %}
+{% set version = "1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+build:
+  noarch: generic
+  number: 0
+
+outputs:
+  - name: sysroot_osx-64
```

Comment:
maybe the name should be `macos-sdk` or similar?

Reply:
Current suggestion matches the naming from https://github.com/conda-forge/linux-sysroot-feedstock/blob/main/recipe/meta.yaml#L146 at least (though there are important parallels that don't hold, so I'm not sure we want to match that naming; e.g. the linux sysroot actually puts files in the env; the osx one is just a metapackage for now).

Reply:
Would stick with the `linux-sysroot` naming. That we can add `osx-arm64` easily here. Or more generally use things like `target_platform` in `name`s

Reply:
```suggestion
  - name: sysroot_{{ cross_target_platform }}
```